xml/System.Threading.Channels/Channel`2.xml

```diff
@@ -39,7 +39,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Initializes an instance of the <see cref="T:System.Threading.Channels.Channel`2" /> abstract class.</summary>
```

Comment:
@rpetrusha should I also remove the `abstract` word from this one?

Reply:
Yes, @carlossanlop. Sorry that I missed it.

Reply:
No worries, I can add it:

```suggestion
        <summary>Initializes an instance of the <see cref="T:System.Threading.Channels.Channel`2" /> class.</summary>
```